front-end/src/api/Api.ts

```diff
@@ -1,33 +1,28 @@
-import axios, { AxiosResponse } from "axios";
+import axios, { AxiosResponse } from 'axios';
 
-export const get = async <T>(
-  endpoint: string,
-  token: string
-): Promise<AxiosResponse<T, any>> => {
-  const response = await axios.get<T>(
-    `http://localhost:3001/api/v1${endpoint}`,
-    {
-      headers: {
-        Authorization: `Bearer ${token}`,
-      },
-    }
-  );
-  return response;
+export const get = async <T>(endpoint: string, token: string): Promise<AxiosResponse<T, any>> => {
+	const response = await axios.get<T>(`http://localhost:3001/api/v1${endpoint}`, {
+		headers: {
+			Authorization: `Bearer ${token}`,
+		},
+	});
+	return response;
 };
 
-export const post = async <T>(
-  endpoint: string,
-  data: T,
-  token: string
-): Promise<AxiosResponse<T, any>> => {
-  const response = await axios.post<T>(
-    `http://localhost:3001/api/v1${endpoint}`,
-    data,
-    {
-      headers: {
-        Authorization: `Bearer ${token}`,
-      },
-    }
-  );
-  return response;
+export const post = async <T>(endpoint: string, data: T, token: string): Promise<AxiosResponse<T, any>> => {
+	const response = await axios.post<T>(`http://localhost:3001/api/v1${endpoint}`, data, {
+		headers: {
+			Authorization: `Bearer ${token}`,
+		},
+	});
+	return response;
+};
+
+export const del = async <T>(endpoint: string, token: string): Promise<AxiosResponse<T, any>> => {
+	const response = await axios.delete<T>(`http://localhost:3001/api/v1${endpoint}`, {
```

Comment:
It would be better to use constant strings with endpoints instead of hardcoding them in multiple places, could be refactoring task for the future

Reply:
Great point! Would you be able to create a task in Jira?